compositor_render/src/renderer.rs

```diff
@@ -176,8 +177,11 @@ impl WgpuCtx {
         let (device, queue) = pollster::block_on(adapter.request_device(
             &wgpu::DeviceDescriptor {
                 label: Some("Video Compositor's GPU :^)"),
-                limits: Default::default(),
-                features: wgpu::Features::TEXTURE_BINDING_ARRAY,
+                limits: wgpu::Limits {
+                    max_push_constant_size: 128,
```

Comment:
no real reason for 128 specifically, for current implementation just 4 would be enough